### DISCUSSION TITLE
Fan Speed Cards cant be controlled.

### DISCUSSION THREAD

**Discussion by KR99999** [Rem0o/FanControl.Releases#794] [Q&A]
Hello,

I have two sets of Lian-Li Uni Sl120 mounted. Each set is made up of 3 fans daisy chained to each other. Both sets are wired directly to the MB. Both sets can be seen in the Speeds view, but I cannot get them into the control panel. Any ideas would be appreciated.

I am using HWinfo for the two sets (Lian-Li Bottom and Lian-Li Side). I have tried assisted setup numerous times without success. Identify fans works and allows me to ramp up or down the rpm's, I just cant get it to be part of the mixed curve.  BIOS has both set to PWM using Aorus Master 570.

Thanks in advance
![FanControl](https://user-images.githubusercontent.com/90014574/168723082-f0891e85-b7e3-4bd1-8296-5ed0def86c9f.jpg)

**Comment by Rem0o** [comment]
HWInfo fans from the plugin are read-only and can't be controlled.

**Reply by KR99999** [reply]
Hi Rem0o, thank you for the reply. That makes sense that it is read only from HWinfo.

I can see (not displayed) the same readouts, not using HWInfo, direct from the MB and it is still greyed out. Any idea how I can make these cards not read only?

Thanks again for the reply

**Reply by Wolphin5** [reply]
Since they are straight to the MB headers... Seems strange you can't control them.  

If they were added after the FanControl was configured, they may just be hidden controls which weren't in use then.  Hit the 3 lines in the top left, and check the slow hidden cards.  Then the 3-dots on each, disable the hidden.
